inspectit-gepard-agent/src/main/java/rocks/inspectit/gepard/agent/configuration/http/HttpConfigurationFactory.java

```diff
@@ -23,8 +26,32 @@ private HttpConfigurationFactory() {}
    */
   public static SimpleHttpRequest createConfigurationRequest(String baseUrl)
       throws URISyntaxException {
-    URI uri = new URIBuilder(baseUrl + "/agent-configuration").build();
 
-    return SimpleRequestBuilder.get(uri).build();
+    AgentInfo agentInfo = AgentInfo.INFO;
+
+    URI uri = new URIBuilder(baseUrl + "/agent-configuration/" + agentInfo.getAgentId()).build();
+
+    SimpleRequestBuilder requestBuilder = SimpleRequestBuilder.get(uri);
+
+    return buildRequestWithHeaders(requestBuilder, agentInfo.getAgent()).build();
+  }
+
+  private static SimpleRequestBuilder buildRequestWithHeaders(
+      SimpleRequestBuilder requestBuilder, Agent agent) {
+    Map<String, String> headers =
+        Map.of(
+            "x-gepard-service-Name", agent.getServiceName(),
+            "x-gepard-vm-id", agent.getVmId(),
+            "x-gepard-gepard-version", agent.getGepardVersion(),
+            "x-gepard-otel-version", agent.getOtelVersion(),
+            "x-gepard-java-version", agent.getJavaVersion(),
+            "x-gepard-start-time", agent.getStartTime().toString());
```

Comment:
These seem like important constants. Can you make them a `public static final String` so we can reuse them in other classes?

Reply:
or tests